server/src/events.mjs

```diff
@@ -24,9 +24,25 @@ import * as stateManagement from './stateManagement.mjs';
 import * as userManagement from './userManagement.mjs';
 import { eventTriggers } from './triggers.mjs';
 import { logger } from './logger.mjs';
-import * as fireboltOpenRpc from './fireboltOpenRpc.mjs'
+import * as fireboltOpenRpc from './fireboltOpenRpc.mjs';
 import { config } from './config.mjs';
 
+function logSuccess(onMethod, result, msg) {
+  logger.info(
+    `${msg}: Sent event ${onMethod} with result ${JSON.stringify(result)}`
+  );
+};
+
+function logErr(onMethod) {
+  logger.info(
```

Comment:
can this be changed to logger.error?

Reply:
Earlier also this was logger.info only, so if we want to change then should be checked with Mike.